src/client/src/apps/SimpleLauncher/applicationConfigurations.tsx

```diff
@@ -95,11 +95,10 @@ const excelLegacyAppConfig: ApplicationConfig = {
 const excelAppConfig = EXCEL_ADAPTER_NAME === 'JS' ? excelJSAppConfig : excelLegacyAppConfig
 
 const env = getEnvironment() || 'unknown'
-const envFormatted = `(${env.toUpperCase()})`
 
 const baseAppConfigs: ApplicationConfig[] = [
   {
-    name: `Reactive Trader Cloud (${envFormatted})`,
+    name: `Reactive Trader Cloud`,
```

Comment:
This needs more thought. The app configs here should match the OpenFin config in https://github.com/AdaptiveConsulting/ReactiveTraderCloud/tree/master/src/client/public/config/openfin in terms of name, uuid and URL.

The name displayed in the launcher should be separate from this.

Reply:
Also, we need to ensure consistency with the application name as shown in the title bar (browser tab) when running in the browser. This title is also used as the name of the application when shown in the task bar in Windows.

Please go through each of the cases in the JIRA, and test them. You can run the launcher and RT against different environments in OpenFin by using the relevant npm commands, e.g. `npm run start:openfin:demo-backend`